src/core/p5.Graphics.js

```diff
@@ -359,6 +360,7 @@ p5.Graphics = class extends p5.Element {
  * // the user double-clicks.
  * function doubleClicked() {
  *   pg.remove();
+ *   doubleClickedBool = true;
```

Comment:
Rather than introducing a new variable here, could we instead do something like this?
```suggestion
 *   pg = undefined;
```
That way, we can continue to use the previous `if (pg)` condition.

Reply:
Thanks for the suggestion! I'll implement that change right away.